spec/acceptance/basic_spec.rb

```diff
@@ -117,3 +117,41 @@ class { 'pulpcore':
   end
 
 end
+
+describe 'with content cache enabled' do
+  certdir = '/etc/pulpcore-certs'
+
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-PUPPET
+      class { 'pulpcore':
+        worker_count => 2,
```

Comment:
Is the number of workers relevant for this test? If not, I'd keep it at the default and not specify it.